proxy_client/common_holder.py

```diff
@@ -122,6 +133,40 @@ async def get_holder_list(
                 holder_list.append(await core_api_client.get_holder_account(op_info.address))
         return holder_list
 
+    @classmethod
+    async def get_legacy_holder_list(
+        cls,
+        core_api_client: CoreApiClient,
+        signer_key_list: Sequence[SolPubKey],
+    ) -> Sequence[HolderAccountModel]:
+        holder_list: list[HolderAccountModel] = list()
+        for key in signer_key_list:
+            if key in list(LHA.keys()):
+                # collect holder accounts with status 51 (Holder Deprecated), 31 (Finalized Deprecated)
+                holder_pubkeys = LHA[str(key)].get("51", []) + LHA[str(key)].get("31", [])
+                for holder_pubkey in holder_pubkeys:
+                    holder_addr = SolPubKey.from_raw(holder_pubkey)
+                    holder_list.append(await core_api_client.get_holder_account(holder_addr))
+        return holder_list
+
+    @classmethod
+    async def destroy_holder(
+        cls,
+        core_api_client: CoreApiClient,
```

Comment:
`core_api_client` isn't used in the function.
Please remove it from the argument list.